src/components/PopupWithForm.js

```diff
@@ -6,6 +6,10 @@ export default class PopupWithForm extends Popup {
     this._modalForm = this._popup.querySelector(config.formSelector);
     this._handleFormSubmit = handleFormSubmit;
     this._inputs = [...this._modalForm.querySelectorAll(config.inputSelector)];
+    this._buttonSave = this._modalForm.querySelector(
+      config.submitButtonSelector
+    );
+    this._selfButtonCaption = this._buttonSave.textContent;
```

Comment:
Good job

nice way of fixing the button text

Reply:
thx